rusk-wallet/src/wallet.rs

```diff
@@ -628,6 +628,21 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
 
         Ok(gas_prices)
     }
+
+    /// Get the amount of stake rewards the user has
+    pub async fn get_stake_amount(
```

Comment:
I need the helper function because `get_stake_info` is a method of the `State` which wallet keeps track off. I'll rename the function to `get_stake_reward`

Reply:
and my appologies, I meant `stake_info` not `get_stake_info` in my first comment.

Reply:
please check the contents of `get_stake_reward` function. It requires a helper function because we need to access the state that means we have to be connected to the chain and be online which can be only accessed through the `Wallet` struct so it needs to be a method @moCello

Reply:
I changed the method to use the `stake_info` method now double check and let me know @moCello